perception/alars/auv_detector/auv_detector/KNN_2.py

```diff
@@ -62,13 +67,60 @@ def __init__(self):
         # self.foreground_publisher = self.create_publisher(Image, 'Quadrotor/core/fpcamera/image_foreground', 10)
         # self.detection_publisher = self.create_publisher(Image, 'Quadrotor/core/fpcamera/image_detection', 10)
         self.buoy_pub = self.create_publisher(Float32MultiArray, f"/{self.robot_name}/{ DroneTopics.BUOY_DETECTOR_ESTIMATE_TOPIC}", 10)
+        self.auv_pub = self.create_publisher(Float32MultiArray, f"alars_detection/auv", 10)
+        self.hough_pub = self.create_publisher(Float32MultiArray, f"alars_detection/hough_circle", 10)
```

Comment:
hard-coded topics. use the topics message. if this file is not used anymore, remove it.